lib/hawkular/inventory/inventory_api.rb

```diff
@@ -20,21 +22,9 @@ def initialize(entrypoint = nil, credentials = {}, options = {})
       @entrypoint = entrypoint
       super(entrypoint, credentials, options)
       version = fetch_version_and_status['Implementation-Version']
-      major, minor = version.scan(/\d+/).map(&:to_i)
-      @entrypoint << '/deprecated' unless major == 0 && minor < 17
-    end
-
-    def fetch_version_and_status
-      if @entrypoint.end_with? '/deprecated'
-        begin
-          backup_entrypoint = @entrypoint
-          @entrypoint = @entrypoint[0...-'/deprecated'.size]
-          return http_get('/status')
-        ensure
-          @entrypoint = backup_entrypoint
-        end
-      end
-      http_get('/status')
+      @version = version.scan(/\d+/).map(&:to_i)
```

Comment:
It's being used, for instance [here](https://github.com/hawkular/hawkular-client-ruby/pull/113/files#diff-2f01ee789318dc3149af508f5d2947d7R20), [here](https://github.com/hawkular/hawkular-client-ruby/pull/113/files#diff-2f01ee789318dc3149af508f5d2947d7R98), etc.